web/client/plugins/FeedbackMask.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const React = require('react');
+const {connect} = require('react-redux');
+const {compose} = require('recompose');
+const {createSelector} = require('reselect');
+const {push} = require('react-router-redux');
+const {Button} = require('react-bootstrap');
+const {get} = require('lodash');
+
+const withMask = require('../components/misc/enhancers/withMask');
+const emptyState = require('../components/misc/enhancers/emptyState');
+const {isLoggedIn} = require('../selectors/security');
+const Message = require('../components/I18N/Message');
+const HTML = require('../components/I18N/HTML');
+
+const feedbackMaskSelector = createSelector([
```

Comment:
just  a simple note, this can go to its selectors.js file? not sure if you reuse it though